internal/distro/rhel85/pipelines.go

```diff
@@ -787,6 +788,157 @@ func ostreePayloadStages(options distro.ImageOptions, ostreeRepoPath string) []*
 	return stages
 }
 
+
+func edgeSimplifiedInstallerPipelines(t *imageType, customizations *blueprint.Customizations, options distro.ImageOptions, repos []rpmmd.RepoConfig, packageSetSpecs map[string][]rpmmd.PackageSpec, rng *rand.Rand) ([]osbuild.Pipeline, error) {
+	pipelines := make([]osbuild.Pipeline, 0)
+	pipelines = append(pipelines, *buildPipeline(repos, packageSetSpecs[buildPkgsKey]))
+	kernelPkg := new(rpmmd.PackageSpec)
+	installerPackages := packageSetSpecs[installerPkgsKey]
+	for _, pkg := range installerPackages {
+		if pkg.Name == "kernel" {
+			kernelPkg = &pkg
+			break
+		}
+	}
+	if kernelPkg == nil {
+		return nil, fmt.Errorf("kernel package not found in installer package set")
+	}
+	imgName := "disk.img"
+	imgNameCompressed := "disk.img.xz"
+	partitionTable := defaultPartitionTable(options, t.arch, rng)
+	kernelVer := fmt.Sprintf("%s-%s.%s", kernelPkg.Version, kernelPkg.Release, kernelPkg.Arch)
+	
+	imageTreePipeline := *simplifiedInstallerImageTreePipeline(repos, installerPackages, &partitionTable, kernelVer, customizations.GetKernel(), t.arch, t.supportsUEFI(), t.kernelOptions, rng, options)
+	pipelines = append(pipelines, imageTreePipeline)
+	imagePipeline := *simplifiedInstallerImagePipeline(imgName, imageTreePipeline.Name, &partitionTable, t.arch)
+	pipelines = append(pipelines, imagePipeline)
+	xzArchivePipeline := *xzArchivePipeline(imagePipeline.Name, imgName, imgNameCompressed)
+	pipelines = append(pipelines, xzArchivePipeline)
+	installerTreePipeline := *simplifiedInstallerImageTreePipeline(repos, installerPackages, &partitionTable, kernelVer, customizations.GetKernel(), t.arch, t.supportsUEFI(), t.kernelOptions, rng, options)
+	pipelines = append(pipelines, installerTreePipeline)
```

Comment:
@achilleas-k @gicmo we said the simplified installer would mimic what the anaconda one does by having the commit pre-built - I'm wondering how to wire that here - any suggestion?

Reply:
This little function here creates a couple of stages for initialising a repository and pulling a commit to prepare the payload on the iso: https://github.com/osbuild/osbuild-composer/blob/53109945efb4ff648982fc6c52af723dede63bd5/internal/distro/rhel85/pipelines.go#L777
The payload stages are fed into the anaconda tree here: https://github.com/osbuild/osbuild-composer/blob/53109945efb4ff648982fc6c52af723dede63bd5/internal/distro/rhel85/pipelines.go#L451

In your case though, it's closer to the tar-installer that includes a tarball for the OS to deploy on install.  That's handled here, by adding these `tarPayloadStages` to the anaconda tree: https://github.com/osbuild/osbuild-composer/blob/53109945efb4ff648982fc6c52af723dede63bd5/internal/distro/rhel85/pipelines.go#L482-L483

But first you'll need to build the raw image from the edge commit, which isn't something we do anywhere else.

Reply:
> But first you'll need to build the raw image from the edge commit, which isn't something we do anywhere else.

isn't this what I've basically done with https://github.com/osbuild/osbuild-composer/pull/1614/files#diff-c427e7b4fcdb45d54447de629f296fb2b63b23cb30170c6f39c2cda118c867a3R861 and https://github.com/osbuild/osbuild-composer/pull/1614/files#diff-c427e7b4fcdb45d54447de629f296fb2b63b23cb30170c6f39c2cda118c867a3R835 (just to triple check, I'm just looking at how to get the base ostree data)

Reply:
Yes, for the live image payload, you'll have to follow the way we create and partition the raw disk and copy the files onto it.  I just meant that there's no pipeline where we specifically do this for edge commits, though I imagine if you have a deployment it's just a matter of copying that tree.

Reply:
can't we just leverage the `--url` from the edge-installer? andbasically do the same, generate the commit, generate the installer with the commit